starter-project-web-vue2/components/aymen/BlogList.vue

```diff
@@ -45,6 +56,10 @@ export default {
   },
   methods: {
     ...mapActions('aymen', ['fetchBlogs', 'deleteBlog']),
+    signout() {
+      localStorage.removeItem("token")
```

Comment:
it's better to not use local storage directly. There are methods in `this.$auth.logout()` or other methods to clear the associated tokens in $auth. since we'll be using it